cluster/common.sh

```diff
@@ -1167,6 +1240,10 @@ function parse-master-env() {
   KUBELET_KEY_BASE64=$(get-env-val "${master_env}" "KUBELET_KEY")
   MASTER_CERT_BASE64=$(get-env-val "${master_env}" "MASTER_CERT")
   MASTER_KEY_BASE64=$(get-env-val "${master_env}" "MASTER_KEY")
+  AGGR_CA_KEY_BASE64=$(get-env-val "${master_env}" "AGGREGATOR_CA_KEY")
+  AGGR_CA_CERT_BASE64=$(get-env-val "${master_env}" "AGGREGATOR_CA_CERT")
```

Comment:
Can we keep the names consistent?

Also AGGREGATOR_CA_CERT should probably be REQUESTHEADER_CA_CERT